Formula/a/async-profiler.rb

```diff
@@ -0,0 +1,56 @@
+class AsyncProfiler < Formula
+  desc "Sampling CPU & HEAP profiler for Java using AsyncGetCallTrace + perf_events"
+  homepage "https://github.com/async-profiler/async-profiler"
+  url "https://github.com/async-profiler/async-profiler/archive/refs/tags/v4.0.tar.gz"
+  sha256 "7beb736868af485d6b0b624e42141f78df0ca8403188adc17965b7153261aa55"
+  license "Apache-2.0"
+  head "https://github.com/async-profiler/async-profiler.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "openjdk" => :test
+
+  def install
+    args = []
+    args << "COMMIT_TAG=#{Utils.git_head}" if build.head?
+    args << "FAT_BINARY=true" if OS.mac?
+    # args << "CC=/usr/local/musl/bin/musl-gcc" if OS.linux?
+
+    system "make", *args, "all"
+
+    bin.install Dir["build/bin/*"]
+    lib.install Dir["build/lib/*"]
+    libexec.install Dir["build/jar/*"]
```

Comment:
No `make install`?

Reply:
I'm not familiar with `make`, but I didn't see any `install` target [there](https://github.com/async-profiler/async-profiler/blob/master/Makefile).

Reply:
might be better to also report the `install` target missing issue